plugins/ms-vscode.node-debug/1.0.0/meta.yaml

```diff
@@ -0,0 +1,10 @@
+id: ms-vscode.node-debug
```

Comment:
Can it be `ms-vscode.node-debug-legacy` and `ms-vscode.node-debug2` just `ms-vscode.node-debug`? Because `ms-vscode.node-debug` and `ms-vscode.node-debug` are confusing names.

Reply:
Now I see that it is name that comes from vscode market place, right?

Reply:
I guess CI will fail. See required fields list https://github.com/eclipse/che-plugin-registry/blob/master/check_plugins_viewer_mandatory_fields.sh#L12

Reply:
@mkuznyetsov Should `firstPublicationDate` and `latestUpdateDate` would be provided automatically or it should be done manually?

It would be nice to see description for `meta.yaml` in REAME.md

Reply:
no need to provide them manually in YAML, as they will be generated during Plugin's Registry dockerimage build